app/app/components/UI/NFTImage/index.js

```diff
@@ -256,20 +256,9 @@ class NFTImage extends PureComponent {
 		return (
 			<FastImage
 				ref={this.refImage}
-				source={
-					defaultLoadingError
-						? defaultImg || require('../../../images/nft_default_placehoder.png')
-						: { uri: urlValue }
-				}
+				source={{ uri: urlValue }}
 				style={[style, showBorder && styles.borderStyle]}
-				onLoadEnd={onLoadEnd}
 				resizeMode={resizeMode}
-				onLoad={onLoad}
```

Comment:
Do we really need to remove the onLoad and onLoadEnd?
Because I don't think they are responsible for the current problem.